src/modules/CourseDetail/components/ReviewForm/index.tsx

```diff
@@ -24,16 +25,20 @@ import { applyEscapedText } from './functions'
 
 const localStorage = new Storage('localStorage')
 
+const { academicYear: defaultAcademicYear } = getCurrentTerm()
+const defaultSemester = '1'
```

Comment:
Shouldn't we use `getPreviousTerm` as a default review term as well?
(So that they can support dynamic term in the future)

Reply:
Do we have that yet?

Reply:
Sorry i mean add it